src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/LegacyReleaseController.cs

```diff
@@ -29,6 +29,14 @@ public async Task<ActionResult<List<LegacyReleaseViewModel>>> ListLegacyReleases
                 .HandleFailuresOrOk();
         }
 
+        [HttpGet("publications/{publicationId:guid}/combined-releases")]
+        public async Task<ActionResult<List<CombinedReleaseViewModel>>> ListCombinedReleases(Guid publicationId)
```

Comment:
I think naming variables `CombinedReleases` is confusing, because it's not immediately clear what has been combined and to what purpose.

I'd be inclined to rename `CombinedReleases` to `ReleaseSeries` and things like `CombinedRelease` to `ReleaseSeriesItem`.

Again, I realise this is a pain to change, but I think the change will be worth it in the long run.

Reply:
Done